job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContext.groovy

```diff
@@ -264,13 +264,21 @@ class WrapperContext extends AbstractExtensibleContext {
     /**
      * Injects globally defined passwords as environment variables into the job.
      *
-     * @since 1.23
+     * @since 1.45
      */
     @RequiresPlugin(id = 'envinject')
-    void injectPasswords() {
```

Comment:
Changing `injectGlobalPasswords` default value to `false` to match plugin's default value is a breaking change and IMHO it's better not to keep this method than to keep the method with same signature but different behaviour.